app/src/examples/ScreenAnimationExample.tsx

```diff
@@ -0,0 +1,100 @@
+// @ts-nocheck
```

Comment:
It will be removed after release of `rn-screens`, because `react-native-screens/gesture-handler` import  currently doesn't exist. We are unable to install the version from GitHub because it doesn't include the generated TypeScript types.